tests/e2e/setup/kafka-wal.yaml

```diff
@@ -26,7 +26,7 @@ metadata:
     strimzi.io/kraft: enabled
 spec:
   kafka:
-    version: 3.7.0
+    version: 3.9.0
```

Comment:
Fixed to: `2024-12-24 07:45:12 WARN  AbstractOperator:566 - Reconciliation #4(watch) Kafka(default/kafka-wal): Failed to reconcile
io.strimzi.operator.cluster.model.KafkaVersion$UnsupportedKafkaVersionException: Unsupported Kafka.spec.kafka.version: 3.7.0. Supported versions are: [3.8.0, 3.8.1, 3.9.0]`